tests/test_tracing.py

```diff
@@ -22,7 +22,8 @@ def test_basic(sentry_init, capture_events, sample_rate):
             pass
 
     if sample_rate:
-        (event,) = events
```

Comment:
I really like this pattern! :P

Reply:
I don't dislike it. The problem was the kind of output we were getting when the test fails "not enough values to unpack" or "too many values to unpack", no clue what the events were.

The fix is adding an assert so that pytest handles printing the contents of `events`. Could have kept the unpacking, but now I don't want to trigger CI again.

Reply:
that's also been a complaint from @mitsuhiko. I might make a pytest plugin that does something similar to assertion rewrites